.github/workflows/cleanspell.pl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/perl
```

Comment:
I am not sure why this is a perl script, would not bash be best?

Reply:
Maybe. I have a feeling this filter is going to grow and get more complicated very soon so a shell script will get quirky very soon.